tensorboard/plugins/debugger/tf_debugger_dashboard/tf-debugger-dashboard.html

```diff
@@ -174,6 +179,9 @@ <h2>Continue...</h2>
         color: black;
         font-weight: bold;
       }
+      #initialDialog {
+        z-index: 102;
```

Comment:
Is there some particular significance to this value?  E.g. does something else have a z-index of 101 or 100?

Reply:
Hmm, this could be smaller. 102 matches the default z-index of backdrops for `paper-dialog`, so I just used that value. I added a comment.